project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/settings/AppSettingsComponent.kt

```diff
@@ -18,7 +18,42 @@ class AppSettingsComponent {
 
     // Fields from MinimizationPluginState
     private val compilationStrategyComboBox = ComboBox(DefaultComboBoxModel(CompilationStrategy.entries.toTypedArray()))
-    private val temporaryProjectLocationField = JBTextField()
+    private val gradleTaskField = JBTextField().apply {
+        emptyText.text = "build"
+    }
+    private val gradleOptionsField = JBTextField().apply {
+        emptyText.text = "--offline, --refresh-dependencies, etc..."
+
+        inputVerifier = object : InputVerifier() {
```

Comment:
Check if my idea with InputVerifier is good.

Reply:
That's not a bad idea to try to verify the input, but there are some concerns:
1. using `,` as a separator: maybe it's better to use just spaces as separators
2. validating options using hard-coded options: it will be better to just use regex to verify (something like `^--[a-zA-Z\\-=]+$`) or use gradle API (not recommended)